tests/per_provider/ci.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2019, Arm Limited, All Rights Reserved
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+# Per provider CI test script
+#
+# This script will execute integration tests targeting a platform with a single provider included.
+# It is meant to be executed inside one of the container which Dockerfiles
+# are in tests/per_provider/provider_cfg/*/.
+#
+# Usage: ./tests/per_provider/ci.sh PROVIDER_NAME
```

Comment:
Might be worth adding a check at the beginning of scripts that *require* an argument and print something like this if that argument wasn't provided

Reply:
Good idea! Will add.